students/wonsuk/westagram/users/views.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from django.core.exceptions import ValidationError
+from django.http            import JsonResponse
+from django.views           import View
+from users.models           import User
+from users.validators       import email_regex_match, password_regex_match
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            
+            email_regex_match(data['email'])
+            password_regex_match(data['password'])
+            
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'message' : 'EMAIL_ALREADY_EXISTS'}, status=401)
+
+            User.objects.create(
+                                username     = data['username'],
+                                email        = data['email'],
+                                password     = data['password'],
+                                phone_number = data['phone_number'],
+                                age          = data['age']
+                                )
```

Comment:
```suggestion
            User.objects.create(
                username     = data['username'],
                email        = data['email'],
                password     = data['password'],
                phone_number = data['phone_number'],
                age          = data['age']
            )
```
줄바꿈 후 에는 4칸만 더 들여써주세요.